### PR TITLE
[PRM-466] Added Initialise Appeal Against The Obligation route

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

File: app/controllers/InitialiseAppealController.scala [hunk 1]
```diff
@@ -60,5 +75,7 @@ class InitialiseAppealController @Inject()(appealService: AppealService,
       .addingToSession((SessionKeys.endDateOfPeriod, appealModel.endDate.toString))
       .addingToSession((SessionKeys.dueDateOfPeriod, appealModel.dueDate.toString))
       .addingToSession((SessionKeys.dateCommunicationSent, appealModel.dateCommunicationSent.toString))
+      .addingToSession((SessionKeys.reasonableExcuse, if (isAppealAgainstObligation) "obligation" else ""))
```
Comment (LewisMitchell) [inline app/controllers/InitialiseAppealController.scala:78]: We have a `isObligationAppeal` session key, and there is some other code that relies on that session key

Comment (LewisMitchell) [reply app/controllers/InitialiseAppealController.scala:78]: Doesn't matter on the value of `isObligationAppeal` just as long as the key is present

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Looks good, just some changes needed

File: app/controllers/InitialiseAppealController.scala (hunk 1 above)
Comment (AndrewRButcher) [reply app/controllers/InitialiseAppealController.scala:78]: Ok missed that will update

Comment (AndrewRButcher) [reply app/controllers/InitialiseAppealController.scala]: Updated based on feedback

**Review by LewisMitchell — APPROVED** [review]
LGTM. AT's pass.

**Pull request merged**